tools/generate-helpers/rocksdb-bindings/rocksdb_plugin.go.tpl

```diff
@@ -0,0 +1,78 @@
+
+package legacy
```

Comment:
Add space below and put this at the top of the file

Reply:
Fixed.